app/controllers/api/v1/orders_controller.rb

```diff
@@ -0,0 +1,66 @@
+class Api::V1::OrdersController < Api::V1::BaseApiController
+  skip_before_action :is_admin?
+  skip_before_action :verify_authenticity_token, :authenticate, only: %i(create update destroy)
+  before_action :find_order, :correct_user_order, only: %i(show update destroy)
+
+  def index
+    render json: Order.includes(:order_details).to_json(include: :order_details)
+  end
+
+  def create
+    total = 0
+    order_params[:order_details_attributes].each do |_k, v|
+      total += (v["price"].to_i * v["quantity"].to_i)
+    end
```

Comment:
```suggestion
    total = 0
    order_params[:order_details_attributes].reduce(0) do |sum, _k, v|
      sum + (v["price"].to_i * v["quantity"].to_i)
    end
```
THử xem có chạy ko em